sdk/face/ai-vision-face-rest/README.md

```diff
@@ -0,0 +1,435 @@
+
+- To create a new Face or Cognitive Services account, you can use [Azure Portal](https://ms.portal.azure.com/#create/Microsoft.CognitiveServicesFace), [Azure PowerShell](https://learn.microsoft.com/azure/ai-services/multi-service-resource?tabs=windows&pivots=azpowershell), or [Azure CLI](https://learn.microsoft.com/azure/ai-services/multi-service-resource?tabs=windows&pivots=azcli).
+
+### Install the package
+
+Install the Azure Face REST client REST client library for JavaScript with `npm`:
+
+```bash
+npm install @azure-rest/ai-vision-face
+```
+
+### Authenticate the client
+
+In order to interact with the Face service, you will need to create an instance of a client.
+An **endpoint** and **credential** are necessary to instantiate the client object.
+
+Both key credential and Microsoft Entra ID credential are supported to authenticate the client.
+For enhanced security, we strongly recommend utilizing Microsoft Entra ID credential for authentication in the production environment, while AzureKeyCredential should be reserved exclusively for the testing environment.
+
+#### Get the endpoint
+
+You can find the endpoint for your Face resource using the Azure Portal or Azure CLI:
+
+```bash
+# Get the endpoint for the Face resource
+az cognitiveservices account show --name "resource-name" --resource-group "resource-group-name" --query "properties.endpoint"
+```
+
+Either a regional endpoint or a custom subdomain can be used for authentication. They are formatted as follows:
+
+```
+Regional endpoint: https://<region>.api.cognitive.microsoft.com/
+Custom subdomain: https://<resource-name>.cognitiveservices.azure.com/
+```
+
+A regional endpoint is the same for every resource in a region. A complete list of supported regional endpoints can be consulted [here](https://azure.microsoft.com/global-infrastructure/services/?products=cognitive-services). Please note that regional endpoints do not support Microsoft Entra ID authentication.
+
+A custom subdomain, on the other hand, is a name that is unique to the Face resource. They can only be used by [single-service resources](https://ms.portal.azure.com/#create/Microsoft.CognitiveServicesFace).
+
+#### Create the client with AzureKeyCredential
+
+To use an API key as the `credential` parameter, pass the key as a string into an instance of [AzureKeyCredential](https://learn.microsoft.com/javascript/api/@azure/core-auth/azurekeycredential?view=azure-node-latest).
+You can get the API key for your Face resource using the [Azure Portal][https://learn.microsoft.com/azure/ai-services/multi-service-resource?tabs=windows&pivots=azportal#get-the-keys-for-your-resource] or [Azure CLI][https://learn.microsoft.com/azure/ai-services/multi-service-resource?tabs=windows&pivots=azcli#get-the-keys-for-your-resource]:
```

Comment:
the format is not correct
![image](https://github.com/Azure/azure-sdk-for-js/assets/9943211/9e7ed6fa-e222-4682-b01b-4f66dc3617f7)